RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -335,7 +335,17 @@ class PumpOpsSynchronous {
 
         pages: for pageNum in 0..<16 {
             NSLog("Fetching page %d", pageNum)
-            let pageData = try getHistoryPage(pageNum)
+            let pageData: NSData
+
+            do {
+                pageData = try getHistoryPage(pageNum)
+            } catch let error as PumpCommsError {
+                if pageNum > 0 {
```

Comment:
So I got back the following (tried it 3 times for good measure): `a7xxxxxx 15 0d 80`

At this point we don't have a `MessageType` entry for `0x15`. I don't know what the `0d` refers to, since this fails when trying to read page 1. 

I'll change this implementation to accept the `0x15` message type and halt further reading.